nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/registry/reflect/MarshallerInspector.java

```diff
@@ -90,6 +91,9 @@
 
     private Object singleton;
 
+    // a a boolean to save the service instrumentation state
+    private boolean servicesInjected = false;
```

Comment:
a a -> a. Use a regular `/** */` comment here. `= false` is the default and should be ommitted.